### PR TITLE
made_mobile_menu_header

### PR DESCRIPTION
**Pull request by Anatolii-Udalov**
Додав файл Html, об'єднав два load у header в index.html. Підключив сss. Виконав мобільне меню хедера (для відкриття дописати is-open).

### CODE REVIEW

File: src/partials/header-menu.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<section class="header-menu-section">
```
Comment (mikk1max) [inline src/partials/header-menu.html]: menu is not a section, just a simple div

File: src/partials/header-menu.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  <nav class="navigation-menu-header-bord">
+    <div class="menu-section-plate">
+      <a class="text-menu-section" href="/src/index.html">yachtjet</a>
+      <button type="button" class="btn-close-header-menu">
+        <svg class="icon-close-header-menu" width="16" height="16">
+          <use href="/img/icons.svg#icon-close"></use>
+        </svg>
+      </button>
+    </div>
+    <ul class="navigation-menu-unnamed-list">
```
Comment (mikk1max) [inline src/partials/header-menu.html]: delete unnamed

File: src/partials/header-menu.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      </li>
+      <li class="list-item-navigation-menu">
+        <a class="navigation-text-header-menu" href="">Yachts</a>
+      </li>
+      <li class="list-item-navigation-menu">
+        <a class="navigation-text-header-menu" href="">Reviews</a>
+      </li>
+    </ul>
+  </nav>
+  <ul class="unnamed-list-media-menu">
```
Comment (mikk1max) [inline src/partials/header-menu.html]: without unnamed, non-logic word

File: src/css/header-menu.css [hunk 4]
```diff
@@ -0,0 +1,157 @@
+.header-menu-section {
+    position: fixed;
+
+    width: 100%;
+    height: 100%;
+    z-index: 999;
```
Comment (mikk1max) [inline src/css/header-menu.css]: we are using only such values (1, 10, 25, 50, 100), don't use 999 ;)

File: src/css/header-menu.css [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    display: flex;
+    align-items: center;
+    border: 1px solid #fbfbfb;
+    border-radius: 8px;
+    padding-right: 20px;
+
+    margin-right: 20px;
+    margin-left: 20px;
+    margin-top: 16px;
+}
```
Comment (mikk1max) [inline src/css/header-menu.css:44]: check final height, it has to equal 42px
Снимок экрана 2024-08-08 в 15 40 26

File: src/css/header-menu.css [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+
+.text-menu-section {
+    font-family: "Inter", sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    letter-spacing: -0.03em;
+    color: #1b414b;
+
+    padding-left: 20px;
```
Comment (mikk1max) [inline src/css/header-menu.css]: make padding left and right in the .menu-section-plate

File: src/css/header-menu.css [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+}
+
+.text-menu-section:active,
+.text-menu-section:hover {
+    color: #055545;
+}
+
+.btn-close-header-menu {
+    height: 28px;
+    width: 28px;
```
Comment (mikk1max) [inline src/css/header-menu.css:61]: svg has to be 28x28 in html markup as an attributes, not the button. Button maybe too, but you will see.

File: src/css/header-menu.css [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+    fill: #0e1823;
+}
+
+.navigation-menu-unnamed-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 4px;
+
+    padding-top: 201px;
```
Comment (mikk1max) [inline src/css/header-menu.css]: at first write 200px and margin, not padding

File: src/css/header-menu.css [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+.navigation-menu-header-bord {
+    margin-bottom: auto;
+
+}
+
+.unnamed-list-media-menu {
+    display: flex;
+    gap: 8px;
+    justify-content: center;
+    margin-bottom: 40px;
```
Comment (mikk1max) [inline src/css/header-menu.css]: don't do margin-bottom, make padding in container

File: src/css/header-menu.css [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+}
```
Comment (mikk1max) [inline src/css/header-menu.css:148]: svg must be 24x24 in html murkup, check on the design
Снимок экрана 2024-08-08 в 15 51 30

**Review by mikk1max — COMMENTED** [review]
Fix some issues, and move on!

**Comment by Anatolii-Udalov** [comment]
Виконав ряд виправлень.

**Review by mikk1max — COMMENTED** [review]
Good job! We'll see what it will look like when all the parts are assembled. I will approve your.

**Review by mikk1max — APPROVED** [review]

**Pull request merged**